src/cdk/table/table.scss

```diff
@@ -1,3 +1,13 @@
 .cdk-table-fixed-layout {
   table-layout: fixed;
 }
+
+.cdk-header-row.cdk-table-sticky,
```

Comment:
Is this useful for non-virtual tables? If not, then maybe add a special css class for virtual

Reply:
Done